packages/angular/create/BUILD.bazel

```diff
@@ -0,0 +1,42 @@
+# Copyright Google Inc. All Rights Reserved.
+#
+# Use of this source code is governed by an MIT-style license that can be
+# found in the LICENSE file at https://angular.io/license
+
+load("//tools:defaults.bzl", "pkg_npm", "ts_library")
+
+licenses(["notice"])  # MIT
+
+package(default_visibility = ["//visibility:public"])
```

Comment:
Nit: Put `//visibility:public` on the targets that actually need to be public (for example `:license` probably doesn't need to be public).

Reply:
Good call. We should update this through out.